_includes/head.html

```diff
@@ -1,7 +1,11 @@
 <head>
   <meta charset='utf-8' />
   <meta http-equiv="X-UA-Compatible" content="chrome=1" />
-  <meta name="description" content="Up For Grabs : Jump in!" />
```

Comment:
Should we keep this tag around (with some more awesome words) for those browsers who don't care for the Open Graph spec?